tests/src/test/scala/whisk/core/cli/test/SwiftTests.scala

```diff
@@ -170,4 +170,79 @@ class SwiftTests
+    it should "allow Swift actions to create a rule" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+
+            // create a trigger name
+            val ruleTriggerName = s"/_/TestTrigger ${System.currentTimeMillis()}"
+
+            // create an action name
+            val ruleActionName = s"/_/TestAction ${System.currentTimeMillis()}"
+
+            // create a name for our rule
+            val ruleName = s"TestRule ${System.currentTimeMillis()}"
+
+            // create an action that creates the trigger which we will use when creating the rule
+            val createTriggerFile = TestUtils.getTestActionFilename("createTrigger.swift")
+            assetHelper.withCleaner(wsk.action, "ActionThatCreatesTrigger") {
+                (action, _) => action.create(name = "ActionThatCreatesTrigger", artifact = Some(createTriggerFile), kind = Some("swift:3"))
+            }
+
+            // invoke the create trigger action
+            val runCreateTrigger = wsk.action.invoke("ActionThatCreatesTrigger", Map("triggerName" -> ruleTriggerName.toJson))
+            withActivation(wsk.activation, runCreateTrigger, initialWait = 5 seconds, totalWait = 60 seconds) {
+                activation =>
+                    // should be successful
+                    activation.response.success shouldBe true
+            }
+
+            // create a dummy action for the rule 
+            val dummyFile = TestUtils.getTestActionFilename("hello.swift")
+            assetHelper.withCleaner(wsk.action, ruleActionName) {
+                (action, _) => action.create(name = ruleActionName, artifact = Some(dummyFile), kind = Some("swift:3"))
+            }
+
+            // create an action that creates the rule
+            val createRuleFile = TestUtils.getTestActionFilename("createRule.swift")
+            assetHelper.withCleaner(wsk.action, "ActionThatCreatesRule") {
+                (action, _) => action.create(name = "ActionThatCreatesRule", artifact = Some(createRuleFile), kind = Some("swift:3"))
+            }
+
+            // invoke the create rule action
+            val runCreateRule = wsk.action.invoke("ActionThatCreatesRule", Map("triggerName" -> ruleTriggerName.toJson, "actionName" -> ruleActionName.toJson, "ruleName" -> ruleName.toJson))
+            withActivation(wsk.activation, runCreateRule, initialWait = 5 seconds, totalWait = 60 seconds) {
+                activation =>
+                    // should be successful
+                    activation.response.success shouldBe true
+
+                    // should have a field named "trigger" which is the name of the trigger associated with the rule
+                    activation.response.result.get.fields("trigger").toString.length should be >= 20
+
+                    // should have a field named "action" which is the name of the action associated with the rule
+                    activation.response.result.get.fields("action").toString.length should be >= 20
+            }
```

Comment:
Yup, manually fire the trigger and then check for activations that show that your rule actually ran the expected action. You can see an example of this here: https://github.com/openwhisk/openwhisk/blob/master/tests/src/test/scala/system/basic/WskRuleTests.scala#L89